Islandora/islandora_defaults
# GitHub actions

**GitHub Issue**: (https://github.com/Islandora/documentation/issues/1798)


# What does this Pull Request do?

Replaces travis with github actions

# What's new?
* removed travis.yml
* added github actions workflow

# How should this be tested?
* watch the github actions go green with this is merged
* note that i ran on this specific branch in my fork (see the results here: https://github.com/asulibraries/islandora_defaults/actions/runs/775077669) and then removed that branch from the github actions configuration. but it should run on 8.x-1.x and PRs of 8.x-1.x once its merged in.


# Interested parties
@Islandora/8-x-committers
